safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -233,6 +252,16 @@ def on_open_output_tool_clicked(self):
             self.tr('Raster file (*.tif)'))
         self.output_path.setText(filename)
 
+    @pyqtSignature('')  # prevents actions being handled twice
+    def on_load_result_toggled(self):
```

Comment:
L89 you are already using 'connect' so I think you don't to use PYQT autoconnect. IMHO, I try to avoid these autoconnects now. I prefer like L89. Just cosmetic too ;-)